Sun_Shengran/progress.md

```diff
@@ -13,44 +13,109 @@ Overall Direction:
 ---
 
 Week 1 Plan:
-- [ ] Build UI frame/structure for displaying of deleted courses and sessions
-- [ ] Create fake deleted items data for the acceptance testing of UI
+- [x] Build UI frame/structure for displaying of deleted courses and sessions
+- [x] Create fake deleted items data for the acceptance testing of UI
+
+Week 1 Work:
+
+- [x] Created PR #8835 [Instructor: recycle bin feature: add instructor recovery page mock up #8834](https://github.com/TEAMMATES/teammates/pull/8835)
+
+Week 1 Retrospect:
+
+This week I built an instructor recovery page and created some fake courses/sessions data for UI testing before the actual development. Since I was new to JSP, it took me some time to figure out how to create a new page and navigate to it by clicking the label on nav bar.
 
 Week 2 Plan:
-- [ ] Add new attribute "deletedDate" for courses and sessions
-- [ ] Modify relevant tests for courses and sessions
+- [x] Add new attribute "deletedDate" for courses and sessions
+- [x] Modify relevant tests for courses and sessions
+
+Week 2 Work:
+
+- NA
+
+Week 2 Retrospect:
+
+This week I created a new attribute "deletedAt" in storage for "Courses" and set the value every time user deleted a course instead of actually deleting it. I also managed to create a Recycle Bin table layout for deleted courses in instructor recovery page. The major difficulty I faced this week is that the new attribute I created cannot be updated, thus the courses deleted by user cannot be shown in Recycle Bin. But later I found out that instead of actually updating the course, it creates a new course and add it directly to the datastore.
```

Comment:
"...and set the value every time user deleted a course instead of actually deleting it" - I think this can be made a little clearer. What value are you talking about?

Reply:
Your last sentence can be clearer as well. What created a new course and added what directly to the datastore?